docs/general/configuration_options.md

```diff
@@ -63,6 +63,10 @@ Possible values:
 - `:singular`
 - `:plural` (default)
 
+##### jsonapi_namespace_separator
+
+Sets separator string for namespaced models to render `type` attribute. Default value is `--`.
+
```

Comment:
Not sure I understand. It's supposed to be a setting for any custom separator, not only dasherized. Why should it be called `dashed_namespace_separator`? The point, that I didn't use `model_name` from rails. Just plain and simple class name processing.